src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1315,5 +1315,73 @@ public void RecordCloneSymbol()
             Assert.Equal(0, clone2.Arity);
             Assert.True(clone2.OverriddenMethod.Equals(clone, TypeCompareKind.ConsiderEverything));
         }
+
+        [Fact]
+        public void AbstractRecordClone()
+        {
+            var src = @"
+abstract record R;
+abstract record R2 : R;
+record R3 : R2;
+record R4 : R2;
+
+class C
+{
+    public static void Main()
+    {
+        R r = new R3();
+        r = r with { };
+    }
+}";
+
+            var comp = CreateCompilation(new[] { src, IsExternalInitTypeDefinition },
+                parseOptions: TestOptions.RegularPreview,
+                options: TestOptions.ReleaseExe);
+
+            var r = comp.GlobalNamespace.GetTypeMember("R");
+            var clone = (MethodSymbol)r.GetMembers(WellKnownMemberNames.CloneMethodName).Single();
+            Assert.False(clone.IsOverride);
+            Assert.False(clone.IsVirtual);
+            Assert.True(clone.IsAbstract);
+            Assert.Equal(0, clone.ParameterCount);
+            Assert.Equal(0, clone.Arity);
```

Comment:
Consider checking display string: `Assert.Equal("... <>Clone(...)", clone.ToTestDisplayString());` in each case.